site/_release/0.10.0.md

```diff
@@ -0,0 +1,604 @@
+* [ARROW-2680](https://issues.apache.org/jira/browse/ARROW-2680) - [Python] Add documentation about type inference in Table.from_pandas
+* [ARROW-2682](https://issues.apache.org/jira/browse/ARROW-2682) - [CI] Notify in Slack about broken builds
+* [ARROW-2689](https://issues.apache.org/jira/browse/ARROW-2689) - [Python] Remove references to timestamps_to_ms argument from documentation
+* [ARROW-2692](https://issues.apache.org/jira/browse/ARROW-2692) - [Python] Add test for writing dictionary encoded columns to chunked Parquet files
+* [ARROW-2695](https://issues.apache.org/jira/browse/ARROW-2695) - [Python] Prevent calling scalar contructors directly
+* [ARROW-2696](https://issues.apache.org/jira/browse/ARROW-2696) - [JAVA] enhance AllocationListener with an onFailedAllocation() call
+* [ARROW-2699](https://issues.apache.org/jira/browse/ARROW-2699) - [C++/Python] Add Table method that replaces a column with a new supplied column
+* [ARROW-2700](https://issues.apache.org/jira/browse/ARROW-2700) - [Python] Add simple examples to Array.cast docstring
+* [ARROW-2701](https://issues.apache.org/jira/browse/ARROW-2701) - [C++] Make MemoryMappedFile resizable
+* [ARROW-2704](https://issues.apache.org/jira/browse/ARROW-2704) - [Java] IPC stream handling should be more friendly to low level processing
+* [ARROW-2713](https://issues.apache.org/jira/browse/ARROW-2713) - [Packaging] Fix linux package builds
+* [ARROW-2717](https://issues.apache.org/jira/browse/ARROW-2717) - [Packaging] Postfix conda artifacts with target arch
+* [ARROW-2718](https://issues.apache.org/jira/browse/ARROW-2718) - [Packaging] GPG sign downloaded artifacts
+* [ARROW-2724](https://issues.apache.org/jira/browse/ARROW-2724) - [Packaging] Determine whether all the expected artifacts are uploaded
+* [ARROW-2725](https://issues.apache.org/jira/browse/ARROW-2725) - [JAVA] make Accountant.AllocationOutcome publicly visible
+* [ARROW-2729](https://issues.apache.org/jira/browse/ARROW-2729) - [GLib] Add decimal128 array builder
+* [ARROW-2731](https://issues.apache.org/jira/browse/ARROW-2731) - Allow usage of external ORC library
+* [ARROW-2732](https://issues.apache.org/jira/browse/ARROW-2732) - Update brew packages for macOS
+* [ARROW-2733](https://issues.apache.org/jira/browse/ARROW-2733) - [GLib] Cast garrow_decimal128 to gint64
+* [ARROW-2738](https://issues.apache.org/jira/browse/ARROW-2738) - [GLib] Use Brewfile on installation process
+* [ARROW-2739](https://issues.apache.org/jira/browse/ARROW-2739) - [GLib] Use G_DECLARE_DERIVABLE_TYPE for GArrowDecimalDataType and GArrowDecimal128ArrayBuilder
+* [ARROW-2740](https://issues.apache.org/jira/browse/ARROW-2740) - [Python] Add address property to Buffer
+* [ARROW-2742](https://issues.apache.org/jira/browse/ARROW-2742) - [Python] Allow Table.from_batches to use Iterator of ArrowRecordBatches
+* [ARROW-2748](https://issues.apache.org/jira/browse/ARROW-2748) - [GLib] Add garrow_decimal_data_type_get_scale() (and _precision())
```

Comment:
Perhaps it's just Github's highlighter, but it seems the leading underscore in `_precision` is making the rest of the text italics.